### ISSUE TITLE
Support crobtan @reboot option

### ISSUE THREAD

**Issue by ludwikbukowski** [sorentwo/oban#212]
### Is your feature request related to a problem? Please describe.

It is possible to schedule cron jobs that are run periodically.
I'd like to have mechanism of running a job for each application startup, only once each time, automatically.
Similarly to [@reboot](https://en.wikipedia.org/wiki/Cron#Overview) option in crontab.

### Describe the Solution You'd Like

We could extend crontab option with reboot, e.g:
```elixir
crontab: [
  {:reboot, MyApp.RebootWorker}
]
```

### Describe Alternatives You've Considered

Currently, in order to run worker on startup, we would have to wait until oban supervision tree is setup and then, somehow insert job. It is not obvious how could we achieve that. It would be nice to have more standardised way.

**Comment by bglusman** [comment]
@sorentwo just looking around at open issues, does this seem like a good ticket I could try and tackle for you, if I have time this weekend perhaps?  Or open to other suggestions, but this makes sense to me and seems (without having read related code yet) probably simple enough.

**Comment by sorentwo** [comment]
@bglusman I'm reworking what is included in the base oban package, so this isn't an optimal thing to contribute to currently. This would be a better first issue https://github.com/sorentwo/oban/issues/210, if you'd like to give that a shot 👍

**Comment by sorentwo** [comment]
Closing in favor of the PR (#268)

**Comment by ludwikbukowski** [comment]
Thank you @sorentwo !
